app/js/ffxivcraftmodel.js

```diff
@@ -338,10 +323,15 @@ function ApplyModifiers(s, action, condition) {
         }
 
         progressIncreaseMultiplier *= nameOfMultiplier;
-        if (s.synth.recipe.aspect !== undefined && s.synth.recipe.aspect == element) {
-            progressIncreaseMultiplier *= 2;
+        if (s.synth.recipe.aspect !== undefined && s.synth.recipe.aspect !== element) {
+            progressIncreaseMultiplier *= 0.5;
         }
     }
+    // Aspected recipes give a global 50% penalty, and using the right brand just negates it
+    else if (s.synth.recipe.aspect !== undefined) {
+        progressIncreaseMultiplier *= 0.5;
+        ftMultiplier = 0.5;
```

Comment:
I'm not sure I understand the math here. Doesn't `ftMultiplier` have to be `2` to negate the `0.5`?